evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -241,11 +241,11 @@ def test_categorical_classification(X_y_categorical_classification):
         y_train=y,
         problem_type="binary",
         objective="precision",
-        max_iterations=5,
+        max_batches=1,
```

Comment:
Updating this test to max_batches = 1, and then checking that none of the cv scores is None. This way, XGBoost is run and if the cv score is none (aka xgboost errors), we catch it here! Run with XGBoost == 1.4.0 to see an error :)

Reply:
So with this change, this test would fail on main right now?

Reply:
@freddyaboulton With this change and xgboost == 1.4.0, this would fail. However, since we don't cap XGBoost, we install >=1.4.2 which passes :)